File: Assignment No_36/Question_2.java
/* 2. Write a java program which accept string from user and count number of small characters.

Input : “Marvellous”
Output : 9

Input : "Ganesh"
Output : 5

*/
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Ass36_2
{
    public static void main(String Arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter The String : ");
        String name = sobj.nextLine();

        int i = 0, iCnt = 0;
        char ch;

        for(i = 0; i < name.length(); i++)
        {
            ch = name.charAt(i);

            if((ch >= 'a') && (ch <= 'z'))
            {
                iCnt++;
            }
        }

        System.out.println("Small characters are : "+iCnt);
    }
}

File: Assignment No_27/Question_2.c
/*2. Write a program which accept string from user and count number of Small characters.

Input : “Marvellous”
Output : 9 

*/
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

int  CountSmall(char *str)
{
	int iCnt=0;

	while(*str !='\0')
	{
	     if(*str>='a' && *str<='z')
	     iCnt++;

	     str++;
	}
	return iCnt;
}

int main()
{
   char Arr[20];
   int iRet=0;

   printf("Enter the string:\n");
   scanf("%[^'\n']s",Arr);

   iRet=CountSmall(Arr);

   printf("Small characters are:%d\t",iRet);

   return 0;
}
